commitlint.config.ts

```diff
@@ -207,6 +207,25 @@ module.exports = {
                         `Please watch out for leading or ending trailing whitespace`
                     ];
                 },
+
+                'prefer-slash-over-backslash': ({header}: {header:any}) => {
```

Comment:
@realmarv please move this plugin to not be the last one (try to make it alphabetical, even though they are not really following proper order now, but at least let's not make it worse :) )